<?xml version="1.0"?>
<launch>
  <node pkg="prob_rob_labs" type="matcher_node" name="matcher_node" output="screen">
  </node>
</launch>
